o2a/converter/oozie_converter.py

```diff
@@ -173,7 +165,7 @@ def read_config_replace_el(self):
 
         :return: None
         """
-        self.config = el_utils.parse_els(properties_file=self.config_file, props=self.props)
+        self.props.config = el_utils.parse_els(properties_file=self.config_file, props=self.props)
```

Comment:
Thanks! nice catch.

Reply:
I think that we should do a separate PR for this. I think it would be useful to delete the config field in this class.

Reply:
It is very confusing.